ci-operator/config/openshift/cluster-api-actuator-pkg/openshift-cluster-api-actuator-pkg-master.yaml

```diff
@@ -95,6 +95,18 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-gcp
+- as: e2e-openstack-operator
+  steps:
+    cluster_profile: openstack-vexxhost
+    test:
+    - as: test
+      cli: latest
+      commands: JUNIT_DIR=${ARTIFACT_DIR} make test-e2e
+      from: stable:cluster-api-actuator-pkg
```

Comment:
the other clauses in this section use `from: src`, why is this one different?

Reply:
because I can't do copy/paste correctly :)
I'll fix it.